components/clarity-repl/src/repl/boot/mod.rs

```diff
@@ -54,28 +52,26 @@ lazy_static! {
     pub static ref POX_3_TESTNET_CODE: String =
         format!("{}\n{}", BOOT_CODE_POX_TESTNET_CONSTS, POX_3_BODY);
     pub static ref BOOT_CODE_COST_VOTING_TESTNET: String = make_testnet_cost_voting();
-    pub static ref STACKS_BOOT_CODE_MAINNET: [(&'static str, &'static str); 11] = [
+    pub static ref STACKS_BOOT_CODE_MAINNET: [(&'static str, &'static str); 10] = [
         ("pox", &BOOT_CODE_POX_MAINNET),
-        ("lockup", BOOT_CODE_LOCKUP),
-        ("costs", BOOT_CODE_COSTS),
-        ("cost-voting", BOOT_CODE_COST_VOTING_MAINNET),
+        ("lockup", &BOOT_CODE_LOCKUP),
+        ("costs", &BOOT_CODE_COSTS),
+        ("cost-voting", &BOOT_CODE_COST_VOTING_MAINNET),
         ("bns", &BOOT_CODE_BNS),
         ("genesis", &BOOT_CODE_GENESIS),
         ("costs-2", &BOOT_CODE_COSTS_2),
-        ("costs-v2", &BOOT_CODE_COSTS_2), // for backwards compatibility with old Clarinet.toml files
```

Comment:
Since we don't support selecting boot contracts in the clarinet.toml anymore, I figured I could remove that